server/channels/api4/role_test.go

```diff
@@ -46,7 +46,7 @@ func TestGetRole(t *testing.T) {
 		Name:          model.NewId(),
 		DisplayName:   model.NewId(),
 		Description:   model.NewId(),
-		Permissions:   []string{"manage_system", "create_public_channel"},
+		Permissions:   []string{"create_direct_channel", "create_public_channel"},
```

Comment:
The "manage_system" permission itself is not being tested here. But since "manage_system" can only be a permission for a SystemAdmin, changing the permissions in the test, just to avoid confusion. Changing the permission does not change the tests.